src/docs/testing/index.md

```diff
@@ -69,7 +69,7 @@ Example:
 Add this file to `test/unit_test.dart`:
 
 {% prettify dart %}
-import 'package:test/test.dart';
+import 'package:flutter_test/flutter_test.dart';
```

Comment:
Overall, I think the current code is fine since we ask folks to import the test package. That said, I'm working on some updates to this page so it references the cookbook a bit more rather than having some duplicated content!